packages/editor/src/store/selectors.js

```diff
@@ -771,7 +761,7 @@ export function isAutosavingPost( state ) {
 	if ( ! isSavingPost( state ) ) {
 		return false;
 	}
-	return !! get( state.saving, [ 'options', 'isAutosave' ] );
+	return state.saving.options?.isAutosave ?? false;
```

Comment:
Should we rather cast to boolean still?

```suggestion
	return Boolean( state.saving.options?.isAutosave );
```

Preferring `Boolean()` over `!!` for readability + reduced risk of errors

Reply:
Good call 👍  Done in https://github.com/WordPress/gutenberg/pull/42502/commits/0ec8c59dcc85209040edad8d5095ed57bfbacf35 and https://github.com/WordPress/gutenberg/pull/42502/commits/46735e8321979986212f6b5e5b6d6bd67cf17145